src/benchmark/java/com/commercetools/sync/benchmark/BenchmarkUtils.java

```diff
@@ -0,0 +1,173 @@
+package com.commercetools.sync.benchmark;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+import java.util.Spliterator;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static java.util.Optional.ofNullable;
+import static java.util.Spliterators.spliteratorUnknownSize;
+import static java.util.stream.StreamSupport.stream;
+
+public class BenchmarkUtils {
+    private static final String BENCHMARK_RESULTS_FILE_NAME = "benchmarks.json";
+    private static final String BENCHMARK_RESULTS_FILE_DIR = ofNullable(System.getenv("TRAVIS_BUILD_DIR"))
```

Comment:
Hardcoding CI specific names (`TRAVIS_BUILD_DIR`) doesn't look flexible.
Better to use our own env name (for instance, `BUILD_DIR`, or `CI_BUILD_DIR`) and assign them in CI configuration

Reply:
a21614f